include/miopen/config.h.in

```diff
@@ -78,11 +78,65 @@
 
 // clang-format off
 #cmakedefine MIOPEN_DEFAULT_FIND_MODE @MIOPEN_DEFAULT_FIND_MODE@
+
 // clang-format on
+#cmakedefine01 MIOPEN_OFFLINE_COMPILER_PATHS_V2
```

Comment:
That is the wrong file for adding those changes. The file is meant to be used in both C and C++ applications. However, the interface header file 'miopen.h' does not need them to work correctly. The compiler paths are internally required; there is no need to be C-compatible. I suggest moving them out of this file and making it C++ only.